app/src/main/java/edu/uc/app/parkfinder/dto/Park.kt

```diff
@@ -2,6 +2,6 @@ package edu.uc.app.parkfinder.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Park(@SerializedName("Code") var code: String, @SerializedName("Name") var name: String){
+data class Park(@SerializedName("Location Number") var code: String, @SerializedName("Location Name") var name: String){
```

Comment:
Changed to work with raw.githubusercontent.com/learn-chef/national-parks-java/master/national-parks JSON endpoint

Reply:
Ok, good, this is what the SerializedName is for.